copter/source/docs/parachute.rst

```diff
@@ -37,39 +37,57 @@ Connecting to the Autopilot
 -  :ref:`CHUTE_SERVO_ON <CHUTE_SERVO_ON>` should be set to the servo position required to release the parachute
 -  :ref:`CHUTE_SERVO_OFF <CHUTE_SERVO_OFF>` should be the "resting" servo position.  I.e. the position the servo is in before the parachute is released
--  set :ref:`SERVO9_FUNCTION <SERVO9_FUNCTION>` = "27".  `SERVO9_` refers to the Pixhawk's AUX OUT 1.  To use AUX OUT2 instead set :ref:`SERVO10_FUNCTION <SERVO10_FUNCTION>` to 27, etc.
--  :ref:`CHUTE_CRT_SINK<CHUTE_CRT_SINK>` should be set to the sink rate in m/s that will trigger the chute release.
+-  set :ref:`SERVO9_FUNCTION <SERVO9_FUNCTION>` = "27". On autopilots with an IOMCU (Outputs labeled MAIN and AUX), ``SERVO9`` refers to the Pixhawk's AUX OUT 1. To use AUX OUT2 instead set :ref:`SERVO10_FUNCTION <SERVO10_FUNCTION>` to 27, etc.
 
-.. note:: :ref:`CHUTE_CRT_SINK<CHUTE_CRT_SINK>` has a bug that will be fixed in future releases which prevents the parachute from being triggered if the vehicle falls in a level attitude, no matter how fast.
+
+Relay Release
+-------------
+
+- Determine/Configure a pin to be a GPIO (see :ref:`common-gpios`).
+- Set that pin number as one of the first four (5 and 6 cannot be used) RELAY functions. ie For GPIO pin 51 using the first RELAY function, set :ref:`RELAY_PIN<RELAY_PIN>` = 51.
+- Since GPIOs are always set low initially during the bootloader period, to avoid accidental release, always use a release mechanism that needs a high output level to trigger and set:
+- :ref:`RELAY_DEFAULT<RELAY_DEFAULT>` = 1 or 2 (low or no change), which determines how RELAY pins are set during the post-bootloader, autopilot initialization period.
+- :ref:`CHUTE_TYPE <CHUTE_TYPE>` = 0,1,2,or 3 to release with RELAY, RELAY2, RELAY3, or RELAY4 functions.
 
 .. image:: ../images/Parachute_MPSetup1.png
     :target: ../_images/Parachute_MPSetup1.png
 
 .. image:: ../images/Parachute_MPSetup2.png
     :target: ../_images/Parachute_MPSetup2.png
 
-Setup Ch7 to manually deploy the parachute from the Mission Planner's
-Extended Tuning page.
+Manual Release
+--------------
+
+You can manually release the parachute using an RC channel/switch: set an ``RCx_OPTION`` to either 22 (Parachute Release) or 23 (Parachute 3Pos). Either function, when the RC channel is high, above 
+:ref:`CHUTE_ALT_MIN<CHUTE_ALT_MIN>`, and is armed. See following section for requirements.
 
-.. image:: ../images/Parachute_MPSetup_Ch7.png
-    :target: ../_images/Parachute_MPSetup_Ch7.png
+RC Disable/Enable of Parachute
+------------------------------
+
+You can disable or enable the parachute AUTOMATIC release using an RC channel/switch: set an ``RCx_OPTION`` to 21. A high enables the automatic release function, low disables it. Manual release is unaffected. If set to 23 (Parachute 3Pos), the low position disables, middle position enables auto operation, a high will attempt to force release, as explained above, assuming the enabling conditions described below.
```

Comment:
I wouldn't capitalise "automatic" but up to you of course.

Reply:
maybe italic or bold lower case....want to differentiate from a manual release...speaking of...you can release with a MAVLINK message also, correct? same conditions apply? ie armed, above min alt, etc.

Reply:
will fix after merge